src/edu/stanford/nlp/util/logging/JavaUtilLoggingAdaptor.java

```diff
@@ -102,16 +102,16 @@ public static void main(String[] args) {
       Redwood.startTrack("Adaptor test controlled by redwood");
 
       Logger topLogger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
-      topLogger.warning("I'm warning you!");
-      topLogger.severe("Now I'm using my severe voice.");
-      topLogger.info("FYI");
+      topLogger.fine("I'm warning you!");
+      topLogger.fine("Now I'm using my severe voice.");
+      topLogger.fine("FYI");
```

Comment:
When logging message mentions log level, the DOI-based approach may not suitable.